// 416p 예외 상황 발생

#include <iostream>

using namespace std;

int main() 
{
	int a = 100, b; 
	cout << "input number: ";
	cin >> b;

	cout << a / b << endl;

	return 0;
}